controllers/provider-gcp/pkg/controller/worker/machines_test.go

```diff
@@ -176,7 +176,7 @@ var _ = Describe("Machines", func() {
 					},
 				}
 
-				w = &extensionsv1alpha1.Worker{
+				wrker = &extensionsv1alpha1.Worker{
```

Comment:
Was the renaming, even with a typo, on purpose?

Reply:
yes, having a one letter variable is quite annoying in tracing, worker is a package, so wrker was the next inline.